.github/workflows/prepare-release.yml

```diff
@@ -2,6 +2,11 @@ name: Prepare Release
 
 on:
   workflow_dispatch:
+    inputs:
+      version:
+        description: Use this to provide a version, instead of deriving it from the changelog.
+        required: false
```

Comment:
If it is not required what will be the version if we don't provide a value here

Reply:
It'll be determined based on the changelog like it used to work before.